Libplanet/Action/NullAction.cs

```diff
@@ -0,0 +1,33 @@
+using Bencodex.Types;
+
+namespace Libplanet.Action
+{
+    /// <summary>
+    /// An action implementation which does nothing for filling type parameter taking of
+    /// <see cref="IAction"/>.
+    /// </summary>
+    public class NullAction : IAction
+    {
+        public NullAction()
+        {
+            PlainValue = default(Null);
```

Comment:
Such expression is no more usable with null checker turned on.  Please use `Null.Value` instead.  See also <https://github.com/planetarium/bencodex.net/issues/20> for details.

```suggestion
            PlainValue = Null.Value;
```